scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowDoFns.scala

```diff
@@ -105,48 +104,92 @@ private[tensorflow] abstract class SavedBundlePredictDoFn[T, V](
 object SavedBundlePredictDoFn {
   def forInput[T, V](
     uri: String,
+    fetchOps: Option[Seq[String]],
     options: TensorFlowModel.Options,
-    fetchOp: Seq[String],
+    signatureName: String,
     inFn: T => Map[String, Tensor[_]],
     outFn: (T, Map[String, Tensor[_]]) => V
-  ): SavedBundlePredictDoFn[T, V] = new SavedBundlePredictDoFn[T, V](uri, options) {
-    override def extractInput(input: T): Map[String, Tensor[_]] = inFn(input)
+  ): SavedBundlePredictDoFn[T, V] = new SavedBundlePredictDoFn[T, V](uri, signatureName, options) {
```

Comment:
@guidj this is a breaking change. I believe a new function should be added to preserve backward compatibility

Reply:
Right, I see